templates/brand/colour-palette.html

```diff
@@ -0,0 +1,707 @@
+{% extends "base.html" %}
```

Comment:
note to reviewer: this page was page was needed as it was missing from this repo, but it is an almost complete rip of the original site, with a few minor changes applied as requested [here](https://github.com/canonical/canonical-design/pull/33#issuecomment-2669150233)